### DISCUSSION TITLE
z(@gnss_coordinate) Now Includes Antenna Height

### DISCUSSION THREAD

**Discussion by DrawQuick** [opengisch/QField#4586] [Bugs]
According to the Qfield documentation the @gnss_coordinate is the value retrieved from the sensor.
"@position_coordinate - A point with the coordinate in WGS84. Lon, Lat, Altitude as delivered by the sensor." "All @position_* variables have a corresponding @gnss_* variable".
This use to be the case, but during one of the recent updates, this has changed. I now get the sensor z value with my antenna height added. I had a field added to store my antenna height using the following expression "z(@gnss_coordinate)-z(@geometry)". This now produces a value of 0 or if I have a vertical grid shift the geoid value.
The value returned from the sensor should not include the antenna height. The antenna height should be stored in an @variable that the user can store. The grid shift value should also be stored as an @variable.
Thank you

**Comment by DrThomasK** [comment]
Maybe related: the height correction does not change anything. If I enter an antenna height of 1,4 m the shown and recorded value is still the same as with no correction at all. I'm using QField 2.8.3 with iPadOS 16.6.1.
